activity/activity.info

```diff
@@ -1,8 +1,7 @@
 [Activity]
 name = Textdungeon
-activity_version = 4
+activity_version = 5
```

Comment:
Changing activity version requires some process, see [maintainer](https://github.com/sugarlabs/sugar-docs/blob/master/src/contributing.md#checklist---maintainer)

Reply:
Yep, sorry, that was stupid